requestor-tutorials/vm-runtime/computation-payload-manifest.md

```diff
@@ -0,0 +1,283 @@
+---
+description: Computation Payload Manifest description, its schema, and configuration guide
+---
+
+# Computation Payload Manifest
+
+_Computation Payload Manifest_ allows [Requestor](../../introduction/requestor.md) to define application package _Payload_ and allows to set constraints on _Computations_ performed on [Provider](../../introduction/provider.md) node.
+
+Request with a manifest can be [configured in yapapi](#configuration). 
+
+Provider node operator controls what computation can be performed by:
+
+  - [Importing certificates](#3-importing-application-authors-certificates) used to sign App authors' certificates into Provider's [keystore](../../provider-tutorials/provider-cli.md#keystore) (which allows to verify _manifest_ signature)
+
+  - Adding domain patterns to Provider's [domain whitelist](../../provider-tutorials/provider-cli.md#domain-whitelist) (which makes [signature](#2-manifest-signature) optional _manifests_)
+
+## Configuration
+
+Manifest can be configured as a [yapapi.payload.vm.manifest](https://yapapi.readthedocs.io/en/latest/api.html#module-yapapi.payload.manifest) function parameter together with optional manifest signature and [App author's certificate](#certificates).
+
+Check [external API request example](../service-development/service-example-6-external-api-request.md) for details.
+
+## Manifest schema
+
+<!-- 
+Schema generated using: https://github.com/golemfactory/yagna/blob/2088-computation-payload-manifest-schema/utils/manifest-utils/README.md#computation-payload-manifest-schema
```

Comment:
I addressed issue in this [Yagna PR comment](https://github.com/golemfactory/yagna/pull/2209#pullrequestreview-1122002269)